### DISCUSSION TITLE
v2/md-enhance/zh/guide/md-import

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#148] [Announcements]
# v2/md-enhance/zh/guide/md-import

VuePress 的 Markdown 增强插件

https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/md-import.html

**Comment by zeanzai** [comment]
在md文档中导入一个md文件时，右侧的TOC找不到了，不知道这是不是一个bug？

**Reply by Mister-Hope** [reply]
它不应该算作一个错误。

因为文档的标题是由@vuepress/markdown 内部提供的extractHeadingPlugin 处理的。

启动项目时，vuepress 会先应用内置的 markdown-it 插件，然后再应用 theme 和 plugins 提供的 markdown-it 插件。 当 mdImport 插件导入并渲染 markdown 内容时，标题已经被 vuepress 提取了。 所以目前没有优雅的方法来解决这个问题。

**Reply by Mister-Hope** [reply]
这个问题已经在最新版本中解决，我们移除了mdImport 并添加了 include 功能，它具有极高的优先级。

**Reply by zeanzai** [reply]
谢谢大佬
